tools/isobuild/meteor-npm.js

```diff
@@ -14,6 +14,7 @@ var buildmessage = require('../utils/buildmessage.js');
 var utils = require('../utils/utils.js');
 var runLog = require('../runners/run-log.js');
 var Profile = require('../tool-env/profile.js').Profile;
+import { version as npmVersion } from 'npm';
```

Comment:
I do prefer this syntax!  But opted for this technique to make the `meteorNpm` API feel more whole (as in, it seemed like the `meteorNpm` api should have a `npmVersion` exposed).